stacks/triliovault-operator/deploy.sh

```diff
@@ -27,8 +32,7 @@ echo "Installing TrilioVault operator and TrilioVault Manager with one-click ins
 
 if [ -z "${MP_KUBERNETES}" ]; then
   # use local version of values.yml
-  ROOT_DIR=$(git rev-parse --show-toplevel)
-  values="$ROOT_DIR/stacks/triliovault-operator/values.yml"
+  values="$TVK_PATH/values.yml"
 else
   # use github hosted master version of values.yml
   values="https://raw.githubusercontent.com/digitalocean/marketplace-kubernetes/master/stacks/triliovault-operator/values.yml"
```

Comment:
The absoulte path of the the license path needs to be given here, otherwise it will fail.

Reply:
@keladhruv I have added the variable here as well. Please check and approve if you think all should work in production now.

Reply:
In the else statement please give the raw github path, otherwise it will fail in production -
`TVK_LICENSE_FILE=https://raw.githubusercontent.com/digitalocean/marketplace-kubernetes/master/stacks/triliovault-operator/tvk_install_license.yaml`